.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,32 +1,45 @@
-# EASI-0000, EASI-0000
-<!-- Follow the pattern of Parent issue, Child issue, if appropriate -->
-
-## Changes and Description
-
-- Change 1
-- Change 2
-
-<!-- Put a description here! -->
-
-## How to test this change
+<!--
```

Comment:
```suggestion
# [EASI-####] [EASI-####] PR TITLE HERE
<!--
```

I think we want to keep the placeholder so each PR starts with the heading

Reply:
I actually kept this out on purpose in favor of using the title of the PR to hold that info

only caveat is that the link isn't clickable in the title, but is when it's in the description 🤔

Reply:
4d0a73335